airflow/traces/tracer.py

```diff
@@ -243,41 +237,52 @@ def start_span_from_taskinstance(
         return EMPTY_SPAN
 
 
-class _Trace(type):
-    factory: Callable
+class _TraceMeta(type):
+    factory: Callable[[], Tracer] | None = None
     instance: Tracer | EmptyTrace | None = None
 
-    def __getattr__(cls, name: str) -> str:
+    def __getattr__(cls, name: str):
```

Comment:
When is this actually invoked? It's anytime someone access `Trace.x` right?

Reply:
Correct